packages/cozy-harvest-lib/src/helpers/oauth.js

```diff
@@ -145,10 +151,19 @@ const getAppSlug = client => {
  * @param  {string} domain    Cozy domain
  * @param  {Object} konnector
  * @param  {string} redirectSlug The app we want to redirect the user on after the end of the flow
- * @return {Object}           Object containing: `oAuthUrl` (URL of cozy stack
- * OAuth endpoint) and `oAuthStateKey` (localStorage key)
+ * @param  {Object} extraParams some extra parameters to add to the query string
+ * @param  {Boolean} reconnect Are we trying to reconnect an existing account ?
+ * @param  {io.cozy.accounts} account targetted account if any
+ * @return {Object}           Object containing: `oAuthUrl` (URL of cozy stack OAuth endpoint) and `oAuthStateKey` (localStorage key)
```

Comment:
```suggestion
 * @return {Object}  oauthurlinfo
  * @returns {string} oauthInfo.oAuthUrl URL of cozy stack OAuth endpoint 
    * @returns {string} oauthInfo.oAuthStateKey URL of cozy stack OAuth endpoint 

```

nit: By the way, shouldn't we use an `URL` object for the `oAuthUrl`? And use built-in method to add params to it instead of concatenating string?